src/neo-vm/ExecutionEngine.cs

```diff
@@ -197,9 +204,20 @@ private bool ExecuteInstruction()
                         return true;
                     }
                 case OpCode.CALL:
+                case OpCode.CALLA:
                     {
+                        int position;
+                        if (instruction.OpCode == OpCode.CALLA)
```

Comment:
Call of a call?

Reply:
What do you mean?

Reply:
There are two options: CALL or CALLA... it's just selecting one of both.
CALLA accepts Pointer, and CALL accepts numeric offset.